Makefile

```diff
@@ -26,7 +26,7 @@ lint: lint-docker lint-hcl lint-bash lint-plugin
 
 .PHONY: lint-docker
 lint-docker:
-	${RUN_CHECK} hadolint
```

Comment:
It looks like you also need to update line 37 as well, since I think lint-bash will do both bash linting and docker linting

Reply:
Good catch.